CMake/FindPackages.cmake

```diff
@@ -58,7 +58,9 @@ common_package(Collage 1.1  REQUIRED )
 common_package(OpenGL   REQUIRED )
 common_package(Boost 1.41.0  REQUIRED COMPONENTS program_options filesystem system thread)
 common_package(Pression 1.0  REQUIRED )
-common_package(X11    )
+if(NOT EQ_AGL_USED)
```

Comment:
I know. It's a placeholder for me, will do properly once your cmake changes are in.